fcc-js/es6/set-default-parameters-for-your-functions/README.md

```diff
@@ -1 +1,28 @@
 # Add challenge details and instructions here
```

Comment:
I think you should remove this, it doesn't have anything that it is doing there, and also the name of the challenge should be the header, not this, You can also apply the (Markdown Guide)[https://www.markdownguide.org/cheat-sheet/]here.